dashboard/test/ui/features/learning_platform/script_overview.feature

```diff
@@ -22,8 +22,13 @@ Feature: Script overview page
     Then I verify progress for lesson 29 level 4 in detail view is "perfect"
     When I click selector ".teacher-panel table td:contains(Sally)" once I see it
     And I wait until element "td:contains(Maze)" is visible
+    # verify name format in summary view
+    And element "td:contains(2. Maze)" is visible
```

Comment:
Not sure I understand how this relates to this change?  Does this verify that we are in summary view?  I see in another test, we wait for `.uitest-summary-cell` vs `.uitest-detail-cell`, would that be more direct?

Reply:
I copied this from another place where we were checking for summary view on script overview, the name of the lesson is formatted like this for summary and not detail view. `uitest-summary-cell` class is used on the teacher dashboard progress table, not on script overview. (As a side note, it's very confusing that the script overview components are called ProgressTables, they don't really feel like tables..) That being said, there's probably a class I could use instead. I was just saving time by copying the pattern I found